app/views/home/index.html.erb

```diff
@@ -1,35 +0,0 @@
-<div class="dgu-masthead">
-  <div class="grid-row dgu-masthead__inner">
-    <div class="column-two-thirds">
-      <h1 class="heading-xlarge">
-        Publish and update data for your organisation
-      </h1>
-      <p class="lede">
-        You must create an account to use this service.
-      </p>
-      <p class="lede">
-        If you already have an account, sign in to publish a new dataset
-        or update an existing one.
-      </p>
-
-      <a class="button dgu-masthead__button-get-started" href="<%= new_user_session_path %>" role="button">
```

Comment:
I've asked @nicholsj about this one too. His steer was that the public facing page should remain and the signon part for now should be limited to changing the "Sign in" button to use the authentication flow provided by Signon instead of our own UI.